base/common/LineReader.h

```diff
@@ -48,7 +48,7 @@ class LineReader
     };
 
     const String history_file_path;
-    static constexpr char word_break_characters[] = " \t\n\r\"\\'`@$><=;|&{(.";
+    static constexpr char word_break_characters[] = " \t\v\f\a\b\r\n`~!@#$%^&*()-=+[{]}\\|;:'\",<.>/?_";
```

Comment:
The only questionable part is `_`. Maybe remove it?

PS. I have merged the code to submodule, we can update it in the same PR.

Reply:
I think it makes sense to keep for consistency with other tools. e.g. psql also treats `_` as a word break char

Reply:
Ok.

Reply:
I guess this PR breaks the completion, after it, if you will type `max_distributed` it will suggest everything that starts from `distributed` instead of only one `max_distributed_connections`

Reply:
@azat Let's remove the underscore character :)

Reply:
@alexey-milovidov will you do this? (creating PR with just this oneline fix looks to me like superfluous CI usage)

Reply:
Yes, I will do.

Creating PR with single line change is Ok.
And I will add a test.